config/prod/JKG-s3.yaml

```diff
@@ -0,0 +1,36 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: GatesKI-TestStudy-Bucket
```

Comment:
is this the correct name for this?  Maybe a better name would be `JKG-Bucket` or something like that?